### ISSUE TITLE
feature: add support for projects with generated code (to support OW2 UC AuthzForce Core)

### ISSUE THREAD

**Issue by slauriere** [STAMP-project/dspot#264]
Trying to run DSpot on [AuthzForce Core](https://gitlab.ow2.org/authzforce/core) with the following properties:

```
#relative path to the project root from dspot project
project=pdp-engine
#relative path to the source project from the project properties
src=src/main/java
#relative path to the test source project from the project properties
testSrc=src/test/java
srcResources=src/main/resources/
testResources=src/test/resources/
#java version used
javaVersion=8
# (optional) path to the output folder, default to "output_diversify"
outputDirectory=dspot-out/
# (optional) filter on the package name containing tests to be amplified ("example" => "example.*"
filter=org.ow2.authzforce*
```

I get the following error:

```
Exception in thread "main" spoon.compiler.ModelBuildingException: org.ow2.authzforce.core.xmlns cannot be resolved to a type at /home/slauriere/OW2/Collabs/STAMP/code/authzforce/core/pdp-engine/src/main/java/org/ow2/authzforce/core/pdp/impl/policy/CoreRefPolicyProviderModule.java:133
	at spoon.support.compiler.jdt.JDTBasedSpoonCompiler.report(JDTBasedSpoonCompiler.java:581)
	at spoon.support.compiler.jdt.JDTBasedSpoonCompiler.reportProblems(JDTBasedSpoonCompiler.java:562)
	at spoon.support.compiler.jdt.JDTBasedSpoonCompiler.build(JDTBasedSpoonCompiler.java:119)
	at spoon.support.compiler.jdt.JDTBasedSpoonCompiler.build(JDTBasedSpoonCompiler.java:100)
	at spoon.Launcher.buildModel(Launcher.java:709)
	at fr.inria.diversify.dspot.support.DSpotCompiler.compile(DSpotCompiler.java:127)
	at fr.inria.diversify.utils.Initializer.initialize(Initializer.java:60)
	at fr.inria.diversify.dspot.DSpot.<init>(DSpot.java:85)
	at fr.inria.stamp.Main.run(Main.java:49)
	at fr.inria.stamp.Main.main(Main.java:32)
```

**Comment by danglotb** [comment]
Hi,

It is because the package `org.ow2.authzforce.core.xmlns` is generated.

I'll have a look to find a solution, and allow **DSpot** to support such project.

Thank you.

**Comment by danglotb** [comment]
This has been fixed with: #342.

Please specify in your properties file: `additionalClasspathElements=<path-to-generated-folder>`
